dockerhost.userdata.tmpl

```diff
@@ -8,4 +16,11 @@ sudo apt update
 sudo apt-add-repository --yes --update ppa:ansible/ansible
 sudo apt install software-properties-common ansible python-apt python-pip -y
 sudo pip install -q jmespath
-ansible-galaxy install -r ansible/requirements.yml
```

Comment:
please add this line back. the ansible playbook will require this role

Reply:
The jumphost is the box from which ansible jobs and similar are executed.

Reply:
and we can't add that line back because this file is executed before the requirements.yml file exists.

Reply:
do we still need to discuss this, or can we resolve the comment?

Reply:
Yes, it can be resolved. Sorry 'bout that.